### DISCUSSION TITLE
Please add some debugging examples after cloning git repo

### DISCUSSION THREAD

**Discussion by ati-ozgur** [quarto-dev/quarto-cli#11045] [Feature Requests]
### What would you like to do?

Report an issue on quarto.org

### Description

There are not examples which shows how to run and debug the current code.
Readme file only shows how to install the quarto-cli from the source code.
But how to debug it is not shown.
vscode workspace also does not provide easy to run commands and debug them.

I would like to run quarto commands and debug it.
For example, I would like to debug following command.

    quarto render --to pdf

**Comment by mcanouil** [comment]
Every documents/projects are different. There is unlikely any example that will suit everyone or even you.

Standard debugging practices consist in removing pieces one at a time until the "code" is reduced to a very small bit that you can easily understand and possibly fix by yourself.

To help you do that, there is a troubleshooting guide: https://quarto.org/docs/troubleshooting/index.html

Anyhow, this still requires for you to trim your project from the unrelated (to your issue) components.

**Reply by ati-ozgur** [reply]
Yeah but the problem is that there is NO example right now.
Apart from compiling the project, you have no examples for others.
With no examples, you could not even start.
